modules/govuk/manifests/apps/ckan.pp

```diff
@@ -146,6 +146,60 @@
         value   => $bulk_worker_processes;
     }
 
+    govuk::app { 'ckan':
+      enable_nginx_vhost => false,
+    }
+
+    govuk::app::nginx_vhost { 'ckan':
+      vhost       => 'ckan',
+      app_port    => $port,
+      # Some API paths expose PII, so we block access by default
+      hidden_paths => ['/api/']
+
+      # Some API paths are required by other apps though, so
+      # safe list them on a case by case basis
+
+      # Used by: datagovuk_publish
+      # Used for: ckan_v26_ckan_org_sync
+      location /api/3/action/organization_list {
+        try_files $uri @app;
+      }
+
+      # Used by: datagovuk_publish
+      # Used for: ckan_v26_ckan_org_sync
+      location /api/3/action/organization_show {
+        try_files $uri @app;
+      }
+
+      # Used by: datagovuk_publish
+      # Used for: ckan_v26_package_sync job
+      location /api/3/search/dataset {
```

Comment:
Not sure why this is coloured differently, wonder if `search` is a keyword that needs escaping by putting in strings? 😬 

Worth trying as-is though!